docker-compose.yml

```
@@ -0,0 +1,53 @@
+
+  backend:
+    build:
+      context: ./backend
+    ports:
+      - "8000:8000"  # Map Django server port
+    volumes:
+      - ./backend:/app  # Mount local code
+    depends_on:
+      - db
+    env_file:
+      - ./backend/.env   #sh -c "python manage.py wait_for_db && python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)